### DISCUSSION TITLE
Could you please add a setting to hide the tips message?

### DISCUSSION THREAD

**Discussion by adrianspacely** [spatie/ray#217] [Ideas]
Could you please add a setting to hide the tips message?

![image](https://user-images.githubusercontent.com/3690710/105111132-59dc1080-5b14-11eb-84b0-a32640d967df.png)

It's not that satisfying clearing the screen when a message pops up.

**Comment by squatto** [comment]
This goes along with [my suggestion to show the release notes on that screen](https://github.com/spatie/ray/discussions/224). Maybe a feature toggle along the lines of "Show tips and release notes on the home screen"?
